pom.xml

```diff
@@ -55,6 +55,7 @@
     <module>pinot-azure-filesystem</module>
     <module>pinot-orc</module>
     <module>pinot-parquet</module>
+    <module>pinot-connectors</module>
```

Comment:
Not related to this part but have a question with the main `pom.xml` file. In my understanding, we currently specify `kafka.version = 0.9.0.1` in the main pom.xml file.  After we add the new connector, each connector module will need to pull different kafka version. It will be good if we can start to think about the issue and how we will be managing that.

Reply:
> Not related to this part but have a question with the main `pom.xml` file. In my understanding, we currently specify `kafka.version = 0.9.0.1` in the main pom.xml file. After we add the new connector, each connector module will need to pull different kafka version. It will be good if we can start to think about the issue and how we will be managing that.

I added some notes in Issue #3998

Reply:
For this iteration itself, I can move it to the pinot-connector-kafka-0.9's pom.xml, something like this: https://github.com/apache/flink/blob/master/flink-connectors/flink-connector-kafka-0.9/pom.xml
What say? @snleee

Reply:
@npawar Does that mean that we can pass `kafka.version` as a property to mvn compilation command and pick default one (0.9 in our case) when it's not given?

If that's the case, it looks good to me 👍

Reply:
This is one step in that direction. For now, I've just removed it from main pom, and kept it only in the connector.pom as a property. Next step will be to figure out a way to inject into it from the main pom